static/src/js/actions/notifications.js

```diff
@@ -0,0 +1,163 @@
+import {
+  PUSHING_NOTIFICATION,
+  NOTIFICATION_PUSHED,
+  NOTIFICATION_DISMISSED
+} from '../constants/actionTypes'
+import {
+  toastPlacement,
+  toastType
+} from '../constants/enums'
+import { removeError } from './errors'
+
+const notificationDefaults = {
+  appearance: toastType.info,
+  autoDismiss: true,
+  autoDismissTimeout: 15000,
+  placement: toastPlacement.topRight
+}
+
+const addToast = (content, notification) => {
```

Comment:
1. This method needs a comment block, as well as line comments.
2. Please rename the `notification` argument, `options`.
3. This method could be added a util given that it isn't using any Redux functionality. We'd suggest adding it to a new file at `/static/src/js/util/addToast.js`
4. Once this is moved, this file can be removed based on the comment below.